src/hooks/useLoadTheme.js

```diff
@@ -7,7 +7,7 @@ const useLoadTheme = () => {
   const { setCurrentTheme, setCustomTheme } = useContext(AppContext);
 
   useEffect(() => {
-    const currentLsVersion = 1;
+    const currentLsVersion = 2;
```

Comment:
this has never been deployed, is there a need to increase?

Reply:
you're right